build_debian.sh

```diff
@@ -245,7 +245,7 @@ sudo cp files/docker/docker.service.conf $_
 sudo sed -i '/After=/s/$/ containerd.service/' $FILESYSTEM_ROOT/lib/systemd/system/docker.service
 
 ## Create redis group
-sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f redis
+sudo LANG=C chroot $FILESYSTEM_ROOT groupadd -f -g 1001 redis
```

Comment:
>1001 [](start = 51, length = 4)

I think the motivation is not to fix the gid for `redis`, but fix the gid for `admin` to 1000.
So suggest
1. create a group with gid 1000 and name `$USERNAME`
2. create a group `redis` without specifying gid
3. create user `$USERNAME` with `-g 1000` and also in the groups `sudo,docker,redis`

Reply:
Agree, we need not  fix gid for redis. But we need to **fix gid for "admin" group**. 
At the same time, it does not limit/restrict anything by explicitly specifying gid for redis.

This fix is nearly a no-op for all purposes as all that is required is a *redis* group and we continue to have it as before.

Reply:
can we pin down the admin group more directly?

Reply:
We could. Just a groupadd. Not sure, about any side effects. Just trying to make the change close to no-op. This change is ~= NO-OP

Reply:
Hold on....There could be something simpler

Reply:
Done. Just shifted order of creation.